.github/workflows/pull_requests.yaml

```diff
@@ -69,3 +69,25 @@ jobs:
           PKGER: "./${{ steps.pkger-binaries.outputs.binary }}"
       - name: Lint
         run: make check
+
+  integration-test:
+    runs-on: 'ubuntu-latest'
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Setup Go
+        uses: actions/setup-go@v2
+      - name: Provision Cluster
+        uses: lkingland/kind-action@v1 # use ./hack/allocate.sh locally
```

Comment:
There's been some work done in knative to have similar actions; as a follow-on I would suggest looking at whether you can just use these instead of having your own.

Reply:
Looking forward to evaluating those options, thanks for the head's up.